ocr.py

```diff
@@ -44,3 +49,116 @@ def ocr(self, image_file, show_steps=False):
         features = self.feature_extractor.extract(image_file.image, segments)
         classes = self.classifier.classify(features)
         return classes, segments
+
+
+class SimpleOCR(object):
+    def __init__(self, segmentation="contour", extraction="simple", classification="knn",
+                 grounding="text"):
+        """
+        Makes the OCR process easier by providing all objects required for doing OCR
+        by itself, but allowing the user to choose other types of objects by specifying
+        them as strings in keyword arguments.
+
+        :param segmentation: contour, rawcontour or raw
+        :param extraction: simple (currently no other options are available)
+        :param classifier: knn (currently no other options are available)
+        :param grounding: text or user
+        """
+        if segmentation == "contour":
```

Comment:
Yesterday, while working on something else, I pretty much the same idea. As it's a class, it would, in my opinion, be better to create a dict as a class variable, but if it's turned into a function anyway changes will have to be made either way.